### ISSUE TITLE
make it easy to validate against specific FPTF type(s)

### ISSUE THREAD

**Issue by ialokim** [public-transport/validate-fptf#4]
Not sure if this is already possible, but I would like to have a way to make sure an object is e.g. a valid `station` object. Perhaps a second argument to `createValidate` listing the type(s) to validate against could do it.

**Comment by derhuerst** [comment]
**To assert that an object is one of one specific type**, use `require(validate-fptf/some-type)`:

```js
const createValidate = require('validate-fptf')
const validateStation = require('validate-fptf/station')

const validate = createValidate()
const s = {type: 'station', id: '123', name: 'foo', location: {latitude: 1, longitude: 2}}
validateStation(validate, s, 'someStation')
```

**To assert that an object is one of several types**, you can use `anyOf`, which currently resides inside `lib`:

```js
const createValidate = require('validate-fptf')
const anyOf = require('validate-fptf/lib/any-od')

const validate = createValidate()
const s = {type: 'station', id: '123', name: 'foo', location: {latitude: 1, longitude: 2}}
anyOf(['stop', 'station'], validate, s, 'someItem')
```

We should probably move `lib/any-of` to the root level, as `lib` usually indicates private stuff.

**Comment by derhuerst** [comment]
The current way to do this is very convoluted. We should make it more straightforward. Would you mind a PR?

**Comment by ialokim** [comment]
> Would you mind a PR?

See #5.

> We should probably move `lib/any-of` to the root level

As I've simplified the way in the PR, there is no need for calling `any-of` from outside anymore.

**Comment by ialokim** [comment]
This has been fixed with 26b752eaaa65b3f487e9e35cc0151a59e7a9afde, 66941263f70a69040ed3b60c44bceff4128f576b and 4c2148cc3ec8e44cbb3700aee5e22937338a869b.
